actinia/location.py

```diff
@@ -106,13 +101,34 @@ def get_mapsets(self):
             self.__request_mapsets()
         return self.mapsets
 
-    # def create_mapset(self, name, epsgcode):
-    #     """
-    #     Creates a location with a specified projection.
-    #     """
-    #     url = f"{self.url}/locations/{name}"
-    #     # TODO
-    #     # resp = requests.post(url, auth=(self.__auth))
+    def create_mapset(self, name):
+        """
+        Creates a mapset with in the location.
+        """
+        mapset = Mapset.create_mapset_request(
+            name,
+            self.name,
+            self.__actinia,
+            self.__auth
+        )
+        mapset.location = self.name
```

Comment:
There is no `location` in the class Mapset, only `__location_name`? And `__location_name` is already set above upon initialization with `self.name`?

Reply:
Nice catch, setting `mapset.location` was from an earlier iteration of the code that I missed when I was cleaning up the code.